lib/provider/LoginProvider.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:async';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/foundation.dart';
+
+
+class LoginProvider {
+  static FirebaseAuth firebaseAuth;
+  static FirebaseFirestore db = FirebaseFirestore.instance;
```

Comment:
firestore instance is used? this class is only for login purpose right ?

Reply:
Yes this page is only for login purpose, added this instance to use it at a later point in time. But now its unnecessary, hence removing this.